test/jest/acceptance/iac/check-path-regression.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { startMockServer } from './helpers';
+
+jest.setTimeout(50000);
+
+describe('checkPath() regression test snyk/cli#3406', () => {
```

Comment:
nit: The paths in the original function's name was in plural.
```suggestion
describe('checkPaths() regression test snyk/cli#3406', () => {
```